x-pack/plugins/apm/public/components/app/backend_detail_overview/backend_throughput_chart.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { asTransactionRate } from '../../../../common/utils/formatters';
+import { useApmBackendContext } from '../../../context/apm_backend/use_apm_backend_context';
+import { useUrlParams } from '../../../context/url_params_context/use_url_params';
+import { useComparison } from '../../../hooks/use_comparison';
+import { useFetcher } from '../../../hooks/use_fetcher';
+import { useTimeRange } from '../../../hooks/use_time_range';
+import { Coordinate, TimeSeries } from '../../../../typings/timeseries';
+import { TimeseriesChart } from '../../shared/charts/timeseries_chart';
+import { useTheme } from '../../../hooks/use_theme';
+
+export function BackendThroughputChart({ height }: { height: number }) {
+  const { backendName } = useApmBackendContext();
+
+  const theme = useTheme();
+
+  const { start, end } = useTimeRange();
+
+  const {
+    urlParams: { kuery, environment },
+  } = useUrlParams();
+
+  const { offset, comparisonChartTheme } = useComparison();
+
+  const { data, status } = useFetcher(
+    (callApmApi) => {
+      if (!start || !end) {
+        return;
+      }
+
+      return callApmApi({
+        endpoint: 'GET /api/apm/backends/{backendName}/charts/throughput',
+        params: {
+          path: {
+            backendName,
+          },
+          query: {
+            start,
+            end,
+            offset,
+            kuery,
+            environment,
+          },
+        },
+      });
+    },
+    [backendName, start, end, offset, kuery, environment]
+  );
+
+  const timeseries = useMemo(() => {
+    const specs: Array<TimeSeries<Coordinate>> = [];
+
+    if (data?.currentTimeseries) {
+      specs.push({
+        data: data.currentTimeseries,
+        type: 'linemark',
+        color: theme.eui.euiColorVis0,
+        title: i18n.translate('xpack.apm.backendThroughputChart.chartTitle', {
+          defaultMessage: 'Throughput',
+        }),
+      });
+    }
+
+    if (data?.comparisonTimeseries) {
+      specs.push({
+        data: data.comparisonTimeseries,
+        type: 'area',
+        color: theme.eui.euiColorMediumShade,
+        title: i18n.translate(
+          'xpack.apm.backendThroughputChart.previousPeriodLabel',
+          { defaultMessage: 'Previous period' }
+        ),
+      });
+    }
+
+    return specs;
+  }, [data, theme.eui.euiColorVis0, theme.eui.euiColorMediumShade]);
```

Comment:
Is this mostly duplicate of https://github.com/elastic/kibana/blob/e771748d96ba098e2e35cf98a010f4f2a23c9c39/x-pack/plugins/apm/public/components/app/service_overview/service_overview_throughput_chart.tsx#L88-L111 ?

Like @cauemarcondes said, would it make sense to re-use more of the existing charts? (I'm all for duplication if the alternative are more complex though).

Reply:
That part is, but lots of other stuff is specific to that instance of the chart (API call, using either the service or backend detail context ,labels). There is common ground, but I'm not sure if an abstraction without generalising the API call as well makes sense. cc @cauemarcondes